src/base/Assert.hh

```diff
@@ -102,33 +104,33 @@
 //! \endcond
 
 #if CELERITAS_DEBUG && defined(__CUDA_ARCH__)
-#    define REQUIRE(COND) CELER_CUDA_ASSERT_(COND)
-#    define CHECK(COND) CELER_CUDA_ASSERT_(COND)
-#    define ENSURE(COND) CELER_CUDA_ASSERT_(COND)
-#    define CHECK_UNREACHABLE CELER_CUDA_ASSERT_(false)
+#    define CELER_EXPECT(COND) CELER_CUDA_ASSERT_(COND)
+#    define CELER_ASSERT(COND) CELER_CUDA_ASSERT_(COND)
+#    define CELER_ENSURE(COND) CELER_CUDA_ASSERT_(COND)
+#    define CELER_ASSERT_UNREACHABLE() CELER_CUDA_ASSERT_(false)
 #elif CELERITAS_DEBUG && !defined(__CUDA_ARCH__)
-#    define REQUIRE(COND) CELER_DEBUG_ASSERT_(COND, precondition)
-#    define CHECK(COND) CELER_DEBUG_ASSERT_(COND, internal)
-#    define ENSURE(COND) CELER_DEBUG_ASSERT_(COND, postcondition)
-#    define CHECK_UNREACHABLE CELER_DEBUG_FAIL_("", unreachable)
+#    define CELER_EXPECT(COND) CELER_DEBUG_ASSERT_(COND, precondition)
+#    define CELER_ASSERT(COND) CELER_DEBUG_ASSERT_(COND, internal)
+#    define CELER_ENSURE(COND) CELER_DEBUG_ASSERT_(COND, postcondition)
+#    define CELER_ASSERT_UNREACHABLE() CELER_DEBUG_FAIL_("", unreachable)
 #else
-#    define REQUIRE(COND) CELER_NOASSERT_(COND)
-#    define CHECK(COND) CELER_NOASSERT_(COND)
-#    define ENSURE(COND) CELER_NOASSERT_(COND)
-#    define CHECK_UNREACHABLE CELER_UNREACHABLE
+#    define CELER_EXPECT(COND) CELER_NOASSERT_(COND)
+#    define CELER_ASSERT(COND) CELER_NOASSERT_(COND)
+#    define CELER_ENSURE(COND) CELER_NOASSERT_(COND)
+#    define CELER_ASSERT_UNREACHABLE() CELER_UNREACHABLE
 #endif
 
 #ifndef __CUDA_ARCH__
-#    define INSIST(COND, MSG) CELER_RUNTIME_ASSERT_(COND, MSG)
+#    define CELER_VALIDATE(COND, MSG) CELER_RUNTIME_ASSERT_(COND, MSG)
 #    define CELER_NOT_CONFIGURED(WHAT) CELER_DEBUG_FAIL_(WHAT, unconfigured)
 #else
-#    define INSIST(COND, MSG)                                                  \
+#    define CELER_VALIDATE(COND, MSG)                                          \
         ::celeritas::throw_debug_error(::celeritas::DebugErrorType::assertion, \
                                        "Insist cannot be called from device "  \
-                                       "code",                                 \
+                                       "from device code",                     \
```

Comment:
Isn't this string concatenate with the previous string and thus the message now reads ".... from device from device code ...." ?

Reply:
Yeah, something messed up perhaps in my inline invocation of clang-format? I'd replaced the macro which reformatted the line thusly, then the line with "Insist" reverted somehow.